File: Searching and Sorting/selection sort.py
#include<iostream>
using namespace std;

void SelectionSort(int a[],int n){
    
    for(int i=0;i<n;i++){
        for(int j=i;j<n;j++){
            if(a[i]>a[j]){
                int temp = a[i];
                a[i] = a[j];
                a[j] = temp;
            }
        }
    }
    
    
    
    }
 

int main(){
    int n;
    cin>>n;
    int a[n];
    for(int i=0;i<n;i++){
        cin>>a[i];
    }
    
    SelectionSort(a,n);

    for(int i=0;i<n;i++){
        cout<<a[i]<<" ";
    }
}

File: VIT SEM 2/MODULE 2/Selection Sort.c
#include <stdio.h>
#include<stdlib.h>

void SelectionSort(int a[],int n){
    int min;
    for(int i=0;i<=n-2;i++){
        min=i;
        for(int j=i+1;j<=n-1;j++){
            if(a[j]<a[min]){
                min = j;
                int temp = a[min];
                a[min] = a[i];
                a[i] = temp;
            }
        }
    }
}
 
int main(){
    int n;
    scanf("%d",&n);
    int a[n];
    for(int i=0;i<n;i++){
        scanf("%d",&a[i]);
    }
    SelectionSort(a,n);
    for(int i=0;i<n;i++){
        printf("%d ",a[i]);
    }
}
